### DISCUSSION TITLE
Is it possible to retrieve GitHub commit status checks?

### DISCUSSION THREAD

**Discussion by MikaelElkiaer** [fluxcd/flux2#4294] [Q&A] [answered]
I was hoping it would be possible to do via:

```bash
gh api "/repos/{owner}/{repo}/commits/$(git rev-parse HEAD)/check-runs"
# or
gh api "/repos/{owner}/{repo}/commits/$(git rev-parse HEAD)/check-suites"
```

but these consequently respond:

```json
{
  "total_count": 0,
  "check_suites": []
}
```

Despite the fact that I can see it in GitHub Web UI:
![2023-10-04-093116_735x146_scrot](https://github.com/fluxcd/flux2/assets/2102306/d1f1c7ed-c094-45b3-89e0-ccc8256ccd9d)

**Comment by makkes** [comment, marked as answer]
First of all, this repository is for Flux discussions, not GitHub API questions so I'd kindly ask you to direct questions on the latter somewhere more appropriate.

Retrieving GitHub commit statuses is documented [here](https://docs.github.com/en/rest/commits/statuses?apiVersion=2022-11-28#get-the-combined-status-for-a-specific-reference).

**Reply by MikaelElkiaer** [reply]
Well, this was a wrong assumption from me in regard to how Flux utilizes GitHub APIs.

I now see the difference between check-runs/-suites and commit statuses.

Thanks.
